rclcpp/include/rclcpp/time_source.hpp

```diff
@@ -74,6 +74,9 @@ class TimeSource
     const rclcpp::QoS & qos = rclcpp::ClockQoS(),
     bool use_clock_thread = true);
 
+  // The TimeSource is uncopyable
+  TimeSource(const TimeSource &) = delete;
```

Comment:
It's not possible to copy the `TimeSource` class already anyway, because it contains a `std::thread` member which is not copyable.

Copy assignment should be deleted, yes, but that doesn't mean that moves should be forbidden. I think it's possible to move all the members so moving the `TimeSource` should be left enabled.

Reply:
Sounds good, yes for the move constructor it seems that defaulting them should be enough

Reply:
@gbiggs could you address this comment (i.e. respecting the rule of 5)?
After that, feel free to run CI (or ping me to run it).

Reply:
Done in 36e05a9b.

@ivanpauno Can you please run CI?